content/docs/releases/enterprise/install/quickstart.mdx

```diff
@@ -12,218 +12,278 @@ import GenerateRecoveryToken from '@site/content/_generate-recovery-token.md';
 
-   ```bash
-   yum -y install pomerium-console
-   ```
+Mount `pgdata` to `/var/lib/postgresql/data` to persist data or to import existing data into your database service and declare the `pgdata` volume outside the database container:
 
-</TabItem>
+```yaml title="database service"
+database:
+  volumes:
+    - ./pgdata:/var/lib/postgresql/data
 
-</Tabs>
+volumes:
+  pgdata:
+```
 
-### System Service
+### Configure the Verify service
 
-Once the package is installed, enable and start the system service:
+Add the configuration for the `verify` service:
 
-```bash
-sudo systemctl enable --now pomerium-console
+```yaml title="verify"
+verify:
+  networks:
+    main: {}
+  image: pomerium/verify:latest
+  expose:
+    - 8000
+  restart: always
 ```
 
-## Initial Configuration
-
-Like the open-source Pomerium base, Pomerium Enterprise is configured through a single config file, located at `/etc/pomerium-console/config.yaml`.
+## Update the Pomerium Core configuration file
 
-### Update Pomerium
+Your `config.yaml` file should look similar to this:
 
-Open your Pomerium config file, `/etc/pomerium/config.yaml`.
+```yaml title="config.yaml"
+authenticate_service_url: https://authenticate.localhost.pomerium.io
 
-1. Add a list item in the `routes` block for the Enterprise Console:
+idp_provider: REPLACE_ME
+idp_client_id: REPLACE_ME
+idp_client_secret: REPLACE_ME
 
-   ```yaml title="/etc/pomerium/config.yaml"
-   routes:
-     - from: https://console.localhost.pomerium.com
-       to: https://localhost:8701
-       policy:
-         - allow:
-             or:
-               - domain:
-                   is: example.com
-       pass_identity_headers: true
-   ```
+# https://pomerium.com/reference/#routes
+routes:
+  - from: https://verify.localhost.pomerium.io
+    to: http://verify:8000
+    policy:
+      - allow:
+          or:
+            - email:
+                is: user@example.com
+    pass_identity_headers: true
+```
 
-   The example value for `to:` assumes Pomerium and Pomerium Enterprise are running on the same test environment.
+Add a [cookie secret](/docs/reference/cookie-secret), [signing key](/docs/reference/signing-key), and [shared secret](/docs/reference/shared-secret):
 
-1. If you haven't already, set `signing_key`. See the [reference page](/docs/reference/signing-key) for more information. The same signing key must be used in both Pomerium Core and Enterprise.
+```yaml
+cookie_secret: REPLACE_ME
+signing_key: REPLACE_ME
+shared_secret: REPLACE_ME
+```
 
-   ```yaml title="/etc/pomerium/config.yaml"
-   signing_key: 'ZZZZZZZZZZZZZZ'
-   ```
+:::tip **Note:**
 
-1. Define the [databroker](/docs/reference/data-broker-service) storage type and connection string. The example below assumes a local Postgres server:
+You must use the same signing key in Pomerium Core and Enterprise.
```

Comment:
suggestion:
~~You must use the same signing key in Pomerium Core and Enterprise.~~
You must use the same signing key and shared secret in Pomerium Core and Enterprise.